### PR TITLE
Updating calico images to the latest release with security fixes

### PR DESCRIPTION
**Pull request by tariq1890**
For more details, please see [Tigera Technical Advisory TTA-2018-001](https://www.projectcalico.org/security-bulletins/)

### CODE REVIEW

File: parts/k8s/addons/1.10/kubernetesmasteraddons-calico-daemonset.yaml [hunk 1]
```diff
@@ -390,7 +390,7 @@ spec:
         # This container installs the Calico CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: quay.io/calico/cni:v3.4.0-0.dev-26-gebf86ed
```
[the remark this answers is not in the document]
Comment (tariq1890) [reply parts/k8s/addons/1.10/kubernetesmasteraddons-calico-daemonset.yaml]: Sure! It should be noted that the article specifically mentions  that it is Calico CNI which has the security issue.

[the remark this answers is not in the document]
Comment (caseydavenport) [reply parts/k8s/addons/1.10/kubernetesmasteraddons-calico-daemonset.yaml]: @jackfrancis @tariq1890 The image here does contain the issue. I should note that the issue will only manifest if one of the following is true:
- log_level set to "debug"
- k8s_auth_token is explicitly set in the CNI config.

Fortunately, neither of those apply in this case, but obviously we should update to a fixed version ASAP.

This version has the logging fix and the Azure glue included: `v3.4.0-0.dev-34-g83daff2`

[the remark this answers is not in the document]
Comment (tariq1890) [reply parts/k8s/addons/1.10/kubernetesmasteraddons-calico-daemonset.yaml]: Thanks for the review @caseydavenport . Making the changes

**Pull request merged**